Machinaris/app.json

```diff
@@ -0,0 +1,10 @@
+{
+    "appid": "e0e4952bfa0738457dac26ef15f46ba9cd49f4c3",
```

Comment:
Where did you get the appid? I don't see an entry for Machinaris at https://apps.heimdall.site/request/list